lib/ansible/modules/cloud/amazon/route53.py

```diff
@@ -565,7 +565,7 @@ def main():
 
     if command_in == 'get':
         if type_in == 'NS':
-            ns = record['values']
+            ns = record['values'] if 'values' in record else []
```

Comment:
Instead, you can make this `record.get('values') or []` instead of the inline conditional.